cmd/gossamer/configcmd_test.go

```diff
@@ -301,11 +304,22 @@ func TestMakeNode(t *testing.T) {
 		{"config", tempFile.Name(), "TOML configuration file", cfgClone},
 	}
 
+	genesispath, err := filepath.Abs("../../genesis.json")
```

Comment:
Lets use tmp file, you might just make something like `createTempGenesis()` similar to `createTempConfigFile()`

Reply:
ok done

Reply:
actually upon further testing I think we need to use an actual genesis or at least one with an actual wasm blob in it, otherwise this test fails when trying to do `makeNode` as it can't instantiate the runtime module

Reply:
ok fixed. it now uses the ext_ test blob